hbw/config/locales/en.yml

```diff
@@ -16,6 +16,7 @@ en:
         cannot_start_process: Cannot start process
         file_list_field_required: Field of "file_list" type with name "homsOrderDataFileList" required
         field_is_required: Field is required
+        task_already_claimed: Task already claimed
```

Comment:
You have task name so you can use substitution with it like `Task %{name} already claimed`.